### ISSUE TITLE
Allow disable directive after a comment

### ISSUE THREAD

**Issue by HaleTom** [koalaman/shellcheck#1267]
#### For new checks and feature suggestions
- [X] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

# Have less highlight source code               # shellcheck disable=SC2034
LESSOPEN="|-$HOME/bin/highlight-less-wrapper"' "%s"'
```

#### Here's what shellcheck currently says:

```
$ shellcheck myscript
 
Line 4:
LESSOPEN="|-$HOME/bin/highlight-less-wrapper"' "%s"'
^-- SC2034: LESSOPEN appears unused. Verify use (or export if used externally).

$ 
```

#### Here's what I wanted or expected to see:

NULL output.

Could disable directives be allowed after a comment?

**Comment by stdedos** [comment]
Instead, directives are _allowed to have_ trailing comments https://github.com/koalaman/shellcheck/issues/1391
